### ISSUE TITLE
Lab-04-Logging Variables

### ISSUE THREAD

**Issue by gzbib** [DS4PS/cpp-528-spr-2021#30]
Hello @cenuno 

I understand that we should avoid logging variables especially if we fail to interpret their coefficients. 

However, in the lab tutorial, I noticed that we logged percent variables to adjust for their skew.  How about this case:

I am using poverty.change variable which is poverty12-poverty00. When I printed the histogram, I got a bell shape, and when I logged the variable, it got skewed. So, can we say that logging a variable is not always a solution? and can we log a variable change in general?

![image](https://user-images.githubusercontent.com/60263729/113521101-108e4f80-95a0-11eb-89d2-4b54f82bbd5b.png)

Thank you in advance.

**Comment by cenuno** [comment]
You are correct, logging is not always the solution. You can log a variable change **only if you are okay with dropping records where the variable change is negative**.

This is direct result of the logging function: https://www.desmos.com/calculator/xtorekpda3. It expects its input to be larger than zero; however, change can be both positive or negative.

This means logging is not the best tool for normalizing your data with variable change.
